package.json

```diff
@@ -4,8 +4,8 @@
   "private": true,
   "scripts": {
     "start": "craco start",
-    "dev": "craco start",
-    "build": "craco build",
+    "dev": "cross-env REACT_APP_VERSION=$(git rev-parse HEAD) craco start",
+    "build": "cross-env REACT_APP_VERSION=$(git rev-parse HEAD) craco build",
```

Comment:
what is this needed for?

Reply:
Its used to have an env variable with the latest commit hash, so we know what exact version of the app is currently deployed.